packages/web/src/types/track/trackType.ts

```diff
@@ -9,11 +9,16 @@ import { ConditionOperators } from '@tee/web/src/types/conditionOperators'
 import type { TrackId } from '@tee/web/src/types'
 
 // FOR TRACKS
-export interface ConditionTrack {
-  type?: string
-  operator?: ConditionOperators
-  value?: any | any[]
-}
+export type ConditionTrack =
+  | {
+      type: string
+      operator: ConditionOperators.is
+      value: any
+    }
+  | {
+      type: string
+      operator: ConditionOperators.exists | ConditionOperators.isMissing
```

Comment:
question: Pourquoi ne pas avoir mis `value?: any` (peut être que je manque quelque chose)

Reply:
Avec cette formulation, une condition avec `ConditionOperators.is` et une propriété "value" manquante renverrait une erreur, ce qui n'est pas le cas avec `value?: any`, donc c'est un poil plus précis. 

Après si ça nuit à la lisibilité et que c'est du fignolage, je peux revenir à `value?: any`, l'enjeu n'est pas très grand :)

Reply:
Ok ! On pourrait peut être voir pour créer des sous types ? (genre `conditionTrackIs`)

Reply:
Bon, j'ai tiré encore un tout petit peu la pelotte, au passage je me suis demandé "pourquoi `ConditionTrack` et pas `Condition` tout court ?"

Il se trouve qu'un autre type `Condition` était défini, utilisé que dans un type, et jamais utilisé en pratique (sns doute un reliquat d'avant publicodes). J'ai retiré tout ça dans [ce commit](https://github.com/betagouv/transition-ecologique-entreprises-widget/pull/388/commits/8d3d21a5a28ae1346fa50e473f17b0727471274a)

Reply:
Praise: Super !! Trop cool !!